lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -48,22 +48,33 @@ def email=(new_email)
       module ClassMethods
         def find_first_by_auth_conditions(tainted_conditions, opts = {})
           filtered_conditions = devise_parameter_filter.filter(tainted_conditions.dup)
-          email = filtered_conditions.delete(:email)
+          criteria = filtered_conditions.extract!(:email, :unconfirmed_email)
```

Comment:
👍 okay, I have 2 questions:
1.  
> criteria.merge!(primary: false, created_at: nil, email: criteria.delete(:unconfirmed_email))

why primary is `false` and created_at is `nil` 

2.  possible to have a primary email that is not confirmed ?